webpack.config.development.js

```diff
@@ -26,17 +26,9 @@ module.exports = {
           },
           {
             loader: 'css-loader',
-            options: {
```

Comment:
Why did you take this out

Reply:
It stopped us from importing our scss directly and we had to put it in a variable like this:
``` import Stlye from './style.scss' ``` which meant our classes on components had to be ``` Style.header ```. I discussed with the team before making changes